packages/block-library/src/template-part/edit/use-template-part-post.js

```diff
@@ -23,14 +23,17 @@ export default function useTemplatePartPost( postId, slug, theme ) {
 			// load the auto-draft created from the
 			// relevant file.
 			if ( slug && theme ) {
```

Comment:
The `theme` value on line 25 seems misleading. We no longer use it in the template parts query for reasons explained in the PR description (it doesn't account for themes that have a parent directory). 

Since we're using the current theme defined by `core/data` to generate a 

It should probably be modified.

Reply:
A follow up makes sense to me, this does seem a bit outdated with how things work now.

Checking the `theme` field that was added to the html markup was an early way to resolve things (and as you have noted a bit problematic when theme directories may be nested elsewhere).  And now that we are only concerned with auto-drafts from currently activated theme using getCurrentTheme instead seems to make sense.